lighthouse-cli/README.md

```diff
@@ -0,0 +1,9 @@
+One time setup:
+
+`npm i -g typescript@2.0 typings`
```

Comment:
yes i will modify so it can all be local, just trying to make it as easy as possible